code/modules/mob/living/simple_animal/friendly/mouse.dm

```diff
@@ -537,9 +544,6 @@
 	mutations = list(M_NO_SHOCK)
 
 /mob/living/simple_animal/mouse/mouse_op/death(var/gibbed = FALSE)
-	..()
+	..(TRUE)
 	if(gibbed == FALSE)
-		src.gib()
-
-/mob/living/simple_animal/mouse/mouse_op/can_be_infected()
```

Comment:
why are you removing this? mouseops are wearing hardsuits, they should not inherit infection behaviors

Reply:
Because simple_animal/mouse checks if infectable, which is false for mouse_op

Reply:
I'm inclined to say keep it in just in case this gets changed back some day, but it's not something you have to do.